lib/hammer_cli_katello.rb

```diff
@@ -43,9 +43,19 @@ def self.exception_handler_class
                                           'hammer_cli_katello/organization'
                                          )
 
-  HammerCLI::MainCommand.lazy_subcommand("gpg", _("Manipulate GPG Key actions on the server"),
+  HammerCLI::MainCommand.lazy_subcommand("gpg",
+                                         _("Manipulate GPG Key actions on the server"),
                                          'HammerCLIKatello::GpgKeyCommand',
-                                         'hammer_cli_katello/gpg_key'
+                                         'hammer_cli_katello/gpg_key',
+                                         :warning =>
+  _('gpg command is deprecated and will be removed in one of the future versions.' \
+    ' Please use content-credentials command instead.')
+                                        )
+
+  HammerCLI::MainCommand.lazy_subcommand("content-credentials",
+                                         _("Manipulate GPG Key actions on the server"),
```

Comment:
GPG keys? Or will this API support non-gpg credentials now or in the future?

Reply:
@akofink I don't know the answer to that question. The original issue was just asking for a command rename. Who would be a good person to ask about the intended direction of that endpoint?

Reply:
I'd say we can go ahead and change it to say something like this (exact wording up to you):

```suggestion
                                         _("Manipulate content credentials (i.e. GPG Keys) on the server"),
```